doc/local-run.md

```diff
@@ -19,4 +19,14 @@ Remove 10 users.
 amoc:remove(10, true).
 ```
 
-NOTE: the exact range of the users can be added using ``amoc_controller:add_users/2`` interface 
\ No newline at end of file
+#### Many independent Amoc nodes
+
+Sometimes it's needed to run several Amoc nodes independently from each other.
+In this case we would like to be able to run different user ranges on every node.
```

Comment:
In this case we would like to be able to run different ranges of user ids on every node.